lib/src/main/java/graphql/nadel/engine/transform/partition/NadelFieldPartitionContext.kt

```diff
@@ -5,6 +5,4 @@ package graphql.nadel.engine.transform.partition
  *
  * An instance is created for each field that is subject to partitioning.
  */
-abstract class NadelFieldPartitionContext {
-    internal object None : NadelFieldPartitionContext()
```

Comment:
There was no use for this object, I'd rather rely on the hook callback to return `null` to indicate no partition is needed for a certain field.

Reply:
I did omit the `None` when I commented in https://github.com/atlassian-labs/nadel/pull/591#discussion_r1802440977 😉